pandas/tests/plotting/frame/test_frame.py

```diff
@@ -760,6 +762,10 @@ def test_plot_scatter_with_c(self):
 
             # n.b. there appears to be no public method
             # to get the colorbar label
+            if mpl_ge_3_4_0:
+                mark = pytest.mark.xfail(reason="_label attribute removed")
```

Comment:
For recent versions, you can use `ax.collections[0].colorbar.ax.get_ylabel()` instead of the private attribute